style.css

```diff
@@ -1,20 +1,25 @@
 body {
-  background-image: url('../mobile-portofolio/image/sabaahmad.png');
-  background-attachment: fixed;
+  background-image: url("/image/picture.png");
```

Comment:
- [ ] Kindly use the correct background image for the mobile version. The current version you are using is the desktop version. Export the correct image on Figma to follow this project requirement as below.
<img width="1277" alt="Screen Shot 2022-08-19 at 8 16 15 AM" src="https://user-images.githubusercontent.com/47673311/185607873-298db59b-bc3e-41ee-b096-de92fa59ca28.png">
Select Header-shapes mobile on the left side and export the file on the right side.